spec/restaurant_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'restaurant'
+
+describe Restaurant do
+  subject(:restaurant) { described_class.new(menu: menu, msg: msg) }
+  let(:msg) { double :msg }
+  let(:menu) { double :menu, view: list }
+  let(:list) { {"Chips" => 3, "Salad" => 4, "Chicken" => 7, "Fish" => 9, "Daily special" => 15} }
```

Comment:
Metrics/LineLength: Line is too long. [97/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)